tests/test_nonbonded.py

```diff
@@ -411,82 +397,70 @@ def test_exclusion(self):
 
         cutoff = 1.0
 
-        for precision, rtol in [(np.float64, 1e-8), (np.float32, 1e-4)]:
+        test_u = potentials.Nonbonded(exclusion_idxs, scales, lambda_plane_idxs, lambda_offset_idxs, beta, cutoff)
+
+        charge_rescale_mask = np.ones((N, N))
```

Comment:
Use nonbonded.convert_exclusions_to_rescale_masks

Reply:
Resolved in [e69becf](https://github.com/proteneer/timemachine/pull/699/commits/e69becf201146c51a3bae9aad88d27ccb3bdc7c0)